rr_rviz_plugins/src/ExamplePanel.cpp

```diff
@@ -0,0 +1,48 @@
+#include <rr_rviz_plugins/ExamplePanel.h>
+#include <QVBoxLayout>
+#include <pluginlib/class_list_macros.h>
+
+/*
+ * Just as in the header, everything needs to happen in the rr_rviz_plugins namespace.
+ */
+namespace rr_rviz_plugins {
+
+ExamplePanel::ExamplePanel(QWidget *parent)
+  : rviz::Panel(parent) // Base class constructor
+{
+  // Panels are allowed to interact with NodeHandles directly just like ROS nodes.
+  ros::NodeHandle handle;
```

Comment:
This section uses 2-space indents while the rest of the PR (and code base) uses 4 spaces

Reply:
Fixed